### DISCUSSION TITLE
Usage of croped images

### DISCUSSION THREAD

**Discussion by IGLOU-EU** [area17/twill#2600] [Q&A] [answered]
Hi there,
I have a newbie question about crops.

In admin interface, I can crop an image ... Ok
The crop seems to working and y have a new entry into the db table mediables with selected area "crop_x, crop_y, crop_w, crop_h"

But, when y want to use it in my template ....
I call `->imagesAsArraysWithCrops('cover')`, that return an array, but every src is the same, so I have tried to use args like `?w={{ $image['width'] }}&h={{ $image['height'] }}` without success. And there are no crops coordinate into the returned array.

```
array(1) {
    [750]=>
    array(3) {
        ["mobile"]=>
        array(6) {
            ["src"]=>
            string(117) "http://127.0.0.1:8000/storage/uploads/fed9e8bf-d67a-4a2d-a2b1-cf11add79bbc/test.jpg"
            ["width"]=>
            int(424)
            ["height"]=>
            int(448)
            ["alt"]=>
            string() ""
            ["caption"]=>
            string(0) ""
            ["video"]=>
            string(0) ""
        }
        ["original"]=>
        array(6) {
            ["src"]=>
            string(117) "http://127.0.0.1:8000/storage/uploads/fed9e8bf-d67a-4a2d-a2b1-cf11add79bbc/test.jpg"
            ["width"]=>
            int(2131)
            ["height"]=>
            int(2249)
            ["alt"]=>
            string(0) ""
            ["caption"]=>
            string(0) ""
            ["video"]=>
            string(0) ""
        }
    }
}
```

Infos:
- PHP 8.3.7
- Twill 2.13.0
- Rendering Service "A17\Twill\Services\MediaLibrary\Loca"

I know I miss something but what ? :disappointed: 
Thanks

**Comment by ifox** [comment, marked as answer]
The local service can't crop images it just serves the original file, you need to use the Glide service.

**Reply by IGLOU-EU** [reply]
Thanks ! I missed the information in doc, I think ? :flushed: 
That can be interesting to not permit cropping when you use local service :thinking:
